src/room/GroupCallErrorBoundary.tsx

```diff
@@ -103,12 +107,14 @@ interface BoundaryProps {
   children: ReactNode | (() => ReactNode);
   recoveryActionHandler?: RecoveryActionHandler;
   onError?: (error: unknown) => void;
+  widget?: WidgetHelpers | null;
```

Comment:
Why is it optional?

Reply:
The helper can be null but you still have to put the prop in the tag, I would have prefer it to be `| undefined`?
But anyhow I just put it like that for convenience in the tests, but that's probably a bad idea